terraform/modules/bb_alarms/variables.tf

```diff
@@ -0,0 +1,135 @@
+variable "vpc_name" {
+  description = "Name of the VPC these alarms are for."
+  type        = "string"
+}
+
+variable "load_balancer_name" {
+  description = "Name of the ELB these alarms are for."
+  type        = "string"
+}
+
+variable "cloudwatch_healthy_hosts_min" {
+  description = "Minimum number of hosts that must be healthy."
+  type        = "string"
+  default     = "1"
+}
+
+variable "cloudwatch_max_latency_secs" {
+  description = "Maximum average latency threshold."
+  type        = "string"
+  default     = "2"
+}
+
+variable "cloudwatch_max_latency_window_secs" {
+  description = "Maximum average latency average window size in seconds."
+  type        = "string"
+  default     = "900"
+}
+
+variable "cloudwatch_notification_arn" {
+  description = "The CloudWatch notification ARN."
+  type        = "string"
+}
+
+variable "app" {}
+variable "env" {}
+
+variable "asg_name" {}
+variable "rds_name" {}
+variable "nat_gw_name" {}
+
+variable "alarm_status_check_failed_enable" {}
+variable "alarm_status_check_failed_eval_periods" {}
+variable "alarm_status_check_failed_period" {}
+variable "alarm_status_check_failed_threshold" {}
+
+variable "alarm_status_check_failed_instance_enable" {}
+variable "alarm_status_check_failed_instance_eval_periods" {}
+variable "alarm_status_check_failed_instance_period" {}
+variable "alarm_status_check_failed_instance_threshold" {}
+
+variable "alarm_status_check_failed_system_enable" {}
+variable "alarm_status_check_failed_system_eval_periods" {}
+variable "alarm_status_check_failed_system_period" {}
+variable "alarm_status_check_failed_system_threshold" {}
+
+variable "alarm_elb_no_backend_enable" {}
```

Comment:
The `alarm_elb_no_backend*`, `alarm_elb_high_latency*`, `alarm_elb_spillover_count*` and `alarm_elb_surge_queue_length*` variables can be removed since they relate to alarms that duplicate what's already included in the `elb_alarms` module.